src/App.js

```diff
@@ -42,6 +43,9 @@ function App() {
       <Route path='/createitem'>
         <CreateItem />
       </Route>
+      <Route path='/updateitem/:id' component={UpdateItem}/>
```

Comment:
Can we create this route using the same syntax as the other routes?

Reply:
We have to use it in this format in order to pass the id.